src/components/loginform/LoginPage.js

```diff
@@ -0,0 +1,47 @@
+import CustomizableAside from "../common/customizableAside";
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar";
+import LoginForm from "./LoginForm";
+import styles from "./loginpage.module.css";
+
+
+function LoginPage() {
+    return(
+        <div className= {styles.login}>
+            {/* aside content */}
+            <div className= {styles.asideContainer}>
+                <CustomizableAside
+                src=  '/image-placeholder.png'
+                width={110}
+                height={110}
```

Comment:
Try not to use 'magic numbers' in your code. Create a const at the top of the file with an appropriate name and use the variable in these places instead.